packages/spectacle/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 9.4.0
+
+### Minor Changes
+
+- Utilize `Kbar` to allow users to quickly search and use the current commands Spectacle supports within presentations. Fixes #1115 ([#1161](https://github.com/FormidableLabs/spectacle/pull/1161))
```

Comment:
Hmmm... It didn't change `#1115` to a link.

Reply:
From [here](https://github.com/svitejs/changesets-changelog-github-compact/blob/main/packages/changesets-changelog-github-compact/README.md), I see this:

> If you use `(see #123)`,`(fix #123)` or `(fixes #123)` in a changeset description, the `#123`will automatically be linkified to the corrosponding issue in your repo

Looks like we might have to explicitly do `(see #1115)` or something of the likes.

Reply:
I went ahead and manually updated the link in this branch, but we might need to do some experimenting and see if we _have_ to do `(see #1115)` for the link-generation to work.

Reply:
When I did mine I had to do the parens, verb then issue number.

Reply:
Oh nice, okay so we've got some triangulation!